sdk/cognitivelanguage/Azure.AI.Language.Conversations/tests/Samples/Sample1_AnalyzeConversationWithOptions.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
There already is a Sample1. You need to do Sample2. The name after "_" doesn't matter.